<launch>
    <node pkg="UWSim" name="UWSim" type="UWSim" 
  args="--configfile $(find ldtravocean)/data/scenes/test_environment.xml 
  --dataPath $(find ldtravocean)/data/scenes"  machine="shoreside"/>
    
    <node pkg="pladypos" type="NavSts2Odom.py" name="stateHat2uwsim" machine="shoreside" >
        <remap from="stateHat" to="rov/meas_sim" />
        <remap from="uwsim_hook" to="uwsim_hook" />
    </node>
</launch>